cli/command/environment_test.go

```diff
@@ -0,0 +1,381 @@
+package command
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/cli/printer"
+	"github.com/quintilesims/layer0/cli/resolver/mock_resolver"
+	"github.com/quintilesims/layer0/client/mock_client"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestCreateEnvironment_userInputErrors(t *testing.T) {
+	_, _, command, _ := initEnvCommandTest(t)
+
+	testCases := []struct {
+		name    string
+		command func(*cli.Context) error
+		args    []string
+	}{
+		{
+			name:    "create",
+			command: command.create,
+		},
+		{
```

Comment:
These are many test cases missing here: one for each missing arg per command, another for invalid ints,  etc.

Reply:
Also, the rest of the open prs have one test per command instead of a single one for all